docs/models.md

```diff
@@ -299,3 +300,30 @@ Only set this when your provider uses a different path or expects query paramete
      }
     }
     ```
+
+### Credential File Authentication
```

Comment:
I want to have this before providers examples, to leave it as last option.

Reply:
Ok, I moved it